assets/javascripts/discourse/initializers/discourse-automation.js

```diff
@@ -28,7 +55,9 @@ function _initializeGLobalUserNotices(api) {
 export default {
   name: "discourse-automation",
 
-  initialize() {
-    withPluginApi("0.8.24", _initializeDiscourseAutomation);
+  initialize(container) {
+    withPluginApi("0.8.24", (api) => {
+      _initializeDiscourseAutomation(api, container);
```

Comment:
Looks like `container` is not used? The `_initializeDiscourseAutomation` signature only has `api` as parameters.